composer.json

```diff
@@ -106,7 +106,11 @@
         "sonata-project/exporter": "^1.7",
         "lightsaml/sp-bundle": "~1.0.3",
         "symfony/expression-language": "~2.8",
-        "egeloen/ordered-form-bundle": "^3.0"
+        "egeloen/ordered-form-bundle": "^3.0",
+
+        "monolog/monolog": "^1.22",
+        "nesbot/carbon": "^1.22",
+        "illuminate/support": "^5.4"
```

Comment:
One of the new dependencies is having problem with PHP 5.6: https://travis-ci.org/mautic/mautic/jobs/246235552#L181